pkg/osbuild/skopeo_source_test.go

```diff
@@ -37,28 +37,28 @@ func TestNewSkopeoSource(t *testing.T) {
 	// empty name
 	expectedErr := regexp.MustCompile(`source item osbuild.SkopeoSourceItem.* has empty name`)
 	assertx.PanicsWithErrorRegexp(t, expectedErr, func() {
-		source.AddItem("", testDigest, imageID, nil)
+		source.AddItem("", testDigest, imageID, nil, nil, nil)
 	})
 
 	// empty digest
 	expectedErr = regexp.MustCompile(`source item osbuild.SkopeoSourceItem.* has invalid digest`)
 	assertx.PanicsWithErrorRegexp(t, expectedErr, func() {
-		source.AddItem("name", "", imageID, nil)
+		source.AddItem("name", "", imageID, nil, nil, nil)
 	})
 
 	// empty image id
 	assert.PanicsWithError(t, `item "" has invalid image id`, func() {
-		source.AddItem("name", testDigest, "", nil)
+		source.AddItem("name", testDigest, "", nil, nil, nil)
 	})
 
 	// invalid digest
 	expectedErr = regexp.MustCompile(`item osbuild.SkopeoSourceItem.* has invalid digest`)
 	assertx.PanicsWithErrorRegexp(t, expectedErr, func() {
-		source.AddItem("name", "foo", imageID, nil)
+		source.AddItem("name", "foo", imageID, nil, nil, nil)
 	})
 
 	// invalid image id
 	assert.PanicsWithError(t, `item "sha256:foo" has invalid image id`, func() {
-		source.AddItem("name", testDigest, "sha256:foo", nil)
+		source.AddItem("name", testDigest, "sha256:foo", nil, nil, nil)
```

Comment:
Would it make sense to test the new validations here as well? Like:
```go
--- a/pkg/osbuild/skopeo_source_test.go
+++ b/pkg/osbuild/skopeo_source_test.go
@@ -61,4 +61,9 @@ func TestNewSkopeoSource(t *testing.T) {
        assert.PanicsWithError(t, `item "sha256:foo" has invalid image id`, func() {
                source.AddItem("name", testDigest, "sha256:foo", nil, nil, nil)
        })
+
+       // invalid image transport
+       assert.PanicsWithError(t, `invalid container transport`, func() {
+               source.AddItem("name", testDigest, testDigest, nil, common.ToPtr("invalid-transport"), nil)
+       })
 }
```
(maybe more, I have not looked too deeply, sorry for that)